File: src/main/java/no/nav/veilarboppfolging/repository/entity/VeilederTilordningEntity.java
package no.nav.veilarboppfolging.repository.entity;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.ZonedDateTime;

@Data
@Accessors(chain = true)
public class VeilederTilordningEntity {
    String aktorId;
    String veilederId;
    boolean oppfolging;
    boolean nyForVeileder;
    ZonedDateTime sistTilordnet;
    ZonedDateTime sistOppdatert;
}

File: src/main/kotlin/no/nav/veilarboppfolging/repository/entity/VeilederTilordningEntity.kt
package no.nav.veilarboppfolging.repository.entity

import java.time.ZonedDateTime

data class VeilederTilordningEntity(
    val aktorId: String,
    val veilederId: String,
    val oppfolging: Boolean,
    val nyForVeileder: Boolean,
    val sistTilordnet: ZonedDateTime?,
    val sistOppdatert: ZonedDateTime?,
)
